src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
If you have the repo url:
```
julia> JSON.parse(HttpCommon.get("https://api.github.com/repos/MichaelHatherly/Docile.jl").data)["description"]
"Julia package documentation system."

julia> (HttpCommon.get("https://api.github.com/users/MichaelHatherly").data |> JSON.parse)["name"]
"Michael Hatherly"
```
and obviously can get `site_name`  from github url too.

:s